scripts/license-header.txt

```diff
@@ -0,0 +1 @@
+Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
```

Comment:
How about placing the license like [iox](https://github.com/influxdata/influxdb_iox)?

Reply:
There is already an license file in root directory, but it's different with header inserted in source code, so I create this new file.